### DISCUSSION TITLE
Default color value for 'ask' color picker?

### DISCUSSION THREAD

**Discussion by jmatthewpryor** [johnlindquist/kit#1080] [Q&A]
I am using 
```
let startColor = await arg({
  type: "color",
  defaultValue: "#eecda3",
});
```
Without success (black is still shown as default color) - any hints about what I am doing wrong, or places to look for more answers
Thanks

**Comment by Ambushfall** [comment]
~~## This is if you'd like to default the value returned~~

~~I'm still figuring out how to default to a certain color in the UI of the color picker itself. (Will update with findings)~~

## Edit 2: I've figured out that input is the default for the PromptData interface used by type color

## Edit 3: Removed all unnecessary code to just address your question

```ts
// Name: default-val

import "@johnlindquist/kit"

let startColor = await arg({
    type: "color",
    input:"#eecda3" // this is where we set the default color
})


inspect(startColor)
```
